README.md

```diff
@@ -31,11 +31,34 @@ inViewport(elem, visible);
 
 function visible(elt) {
   // elt === elem
-  alert(elt.id + ' is visible in the window !');
+  alert(elt.id + ' is visible in the window!');
 }
 ```
 The first callback argument is always the `element` that entered the viewport.
 
+### callback watcher API
+
+The callback is called only one time, when the `element` is in the viewport for the first time.
+At any time you can rewatch or stop watching, by using the `watch` and ` dispose` API. 
```

Comment:
Oh wow good thing we add documentation, I did not get we could rewatch, pretty cool to do something like an infinite scroll simple function.